deploy/kubernetes/elastic-agent-managed-kubernetes.yaml

```diff
@@ -114,26 +105,14 @@ spec:
         - name: varlog
           hostPath:
             path: /var/log
-        # Needed for cloudbeat
-        - name: etc-kubernetes
+        # Needed for Cloud Security Posture integration (cloudbeat)
+        - name: etc-full
           hostPath:
-            path: /etc/kubernetes
-        # Needed for cloudbeat
+            path: /etc
+        # Needed for Cloud Security Posture integration (cloudbeat)
```

Comment:
Maybe having a link to related docs here would be helpful. Also if this is only required for cloudbeat, I would mention that users can remove it if they don't aim using cloudbeat at all.

Reply:
Unfortunately at the moment I can't think of a good public doc that can be linked to. I've added a note about removing it.